### ISSUE TITLE
How to debug Nashorn JavaScript in IntelliJ IDEA

### ISSUE THREAD

**Issue by FishGel** [winterbe/java8-tutorial#37]
I want to debug this project's JavaScript in  IntelliJ IDEA, but is not working.  Can you debug ? @winterbe

**Comment by vin0010** [comment]
Yeah its possible. Only issue I am facing is, intellij not detecting typescript and other front end framework specific scripts. 
https://www.jetbrains.com/help/idea/debugging-javascript-in-chrome.html
This one helped me to do debug JS code in IntelliJ IDEA.
